sros2/setup.py

```diff
@@ -18,10 +19,18 @@ def package_files(directory):
 extra_files.extend(package_files('sros2/policy/templates'))
 
 
+package_name = 'sros2'
+
 setup(
-    name='sros2',
+    name=package_name,
     version='0.6.2',
     packages=find_packages(exclude=['test']),
+    data_files=[
+        ('share/' + package_name, ['package.xml']),
+        ('share/ament_index/resource_index/packages',
+            ['resource/' + package_name]),
```

Comment:
These lines are fixing an unrelated issue-- mind extracting them into their own PR? Thanks for the fix, though!

Reply:
@kyrofa Yeah, I can open a PR for 
9b20326 and then rebase 045699d once it's merged

Reply:
@kyrofa Done https://github.com/ros2/sros2/pull/162

Reply:
@kyrofa Ok, this one is rebased now so it just contains one commit relevant to the XML catalog